client/src/App.js

```diff
@@ -9,18 +10,27 @@ import Login from "./Login";
 import Passwords from "./Passwords";
 import Register from "./Register";
 import SetUp from "./components/NewUser/SetUp";
-
+import Overview from "./components/Overview";
 import BudgetForm from "./components/budget/BudgetForm";
 import Expenses from "./components/expenses/Expenses";
 
 function App() {
+  const [path, setPath] = useState("");
+
+  // useEffect(() => {
+  //   console.log(path);
+  // }, [path]);
+
   return (
     <AuthContextProvider>
       <Router>
-        <Header />
+        {path === "/home" ? "" : <Header />}
```

Comment:
Instead of conditionally showing/hiding the header here, and splitting the header implementation between Header and Home, I recommend always showing the Header, and simply styling the Header appropriately based on which route path is currently active (and whether the user is logged in or not).